SpatialGDK/Source/SpatialGDK/Public/Utils/SpatialStatics.h

```diff
@@ -79,6 +79,13 @@ class SPATIALGDK_API USpatialStatics : public UBlueprintFunctionLibrary
 	UFUNCTION(BlueprintCallable, meta = (WorldContext = "WorldContextObject", CallableWithoutWorldContext, Keywords = "log spatial", AdvancedDisplay = "2", DevelopmentOnly), Category = "Utilities|Text")
 	static void PrintTextSpatial(UObject* WorldContextObject, const FText InText = INVTEXT("Hello"), bool bPrintToScreen = true, FLinearColor TextColor = FLinearColor(0.0, 0.66, 1.0), float Duration = 2.f);
 
+	/**
+	 * Returns true if worker flag was found.
+	   Gets value of a worker flag.
+	 */
+	UFUNCTION(BlueprintCallable, Category = "SpatialOS", meta = (WorldContext = "WorldContextObject"))
+	static bool GetWorkerFlag(const UObject* WorldContextObject, const FString& Name, FString& Value);
```

Comment:
Are you planning to add a blueprint test for this function in the test gym?

Reply:
Yes that's the plan